pkgs/applications/radio/soapyrtlsdr/default.nix

```diff
@@ -1,9 +1,9 @@
 { lib, stdenv, fetchFromGitHub, cmake, pkg-config
-, rtl-sdr, soapysdr
+, rtl-sdr, soapysdr, darwin
```

Comment:
The policy is to include the frameworks in the arguments directly and inherit them in `top-level/all-packages.nix`. You'll find a few examples there.

Reply:
Cool, will have that fixed. Thanks for the quick review!